capella_console_client/assets.py

```diff
@@ -100,7 +100,13 @@ def _gather_download_requests(
 def _get_raster_href(assets_presigned: Dict[str, Any]) -> str:
     raster_asset = assets_presigned.get("HH")
     if raster_asset is None:
-        raster_asset = assets_presigned["VV"]
+        try:
```

Comment:
I think we can simplify this to something along the lines of

```python
# AT TOP OF FILE
MAIN_ASSET_KEY_OPTIONS = {"HH", "VV", "analytic_product"}

# here
try: 
    intersect = set(assets_presigned).intersection(MAIN_ASSET_KEY_OPTIONS)
    main_asset_key = list(intersect)[0]
    raster_asset = assets_presigned[main_asset_key]
except IndexError:
    raise ValueError("none of {', '.join(MAIN_ASSET_KEY_OPTIONS)} found")

Reply:
Done!